example-setup/scripts/install_generic.sh

```diff
@@ -35,8 +35,7 @@ cat <<EOF> /etc/consul.d/consul.json
   "bootstrap_expect": 3,
   "server": true,
   "enable_script_checks": true,
-  "disable_remote_exec": true,
-  "start_join": ["192.168.2.11", "192.168.2.12", "192.168.2.13"]
```

Comment:
its ok this time, but this seems to be an unrelated change. please keep unrelated changes separate from each other :)